app/javascript/modCenter/singleArticle/singleArticle.test.jsx

```diff
@@ -29,15 +47,32 @@ describe('<SingleArticle />', () => {
       />,
     );
 
-  it('should have no a11y  violations', async () => {
+  const renderSingleArticle2 = (article = testArticle2) =>
+    render(
+      <SingleArticle
+        id={article.id}
+        title={article.title}
+        path={article.path}
+        publishedAt={article.publishedAt}
+        cachedTagList={article.cachedTagList}
+        user={article.user}
+      />,
+    );
+
+  it('should have no a11y violations', async () => {
     const { container } = renderSingleArticle();
     const results = await axe(container);
     expect(results).toHaveNoViolations();
   });
 
   it('renders the article title', () => {
     const { getByText } = renderSingleArticle();
-    getByText(testArticle.title);
+    getByText(testArticle1.title);
+  });
+
+  it('truncates the article title if longer than 60 characters', () => {
+    const { getByText } = renderSingleArticle2();
```

Comment:
So for example here you can do `renderSingleArticle(testArticle2)`

Reply:
👍